tests/smartswitch/platform_tests/test_show_platform_dpu.py

```diff
@@ -12,7 +12,7 @@
 from tests.common.devices.sonic import *  # noqa: 403
 
 pytestmark = [
-    pytest.mark.topology('t1')
+    pytest.mark.topology('smartswitch-t1')
```

Comment:
the marker should be 'smartswitch', like pytest.mark.topology('t1') for all t1 topos and pytest.mark.topology('t0') for all t0.

Reply:
Changed it. Since you mentioned to change the yaml file to smartswitch-t1, I used marker as well to be changed. now changed back the marker to just smartswitch.